NightscoutServiceKit/RemoteCommands/Actions/Action.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Action.swift
+//  LoopKit
+//
+//  Created by Bill Gestrich on 12/25/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public enum Action: Codable {
+    case temporaryScheduleOverride(OverrideAction)
+    case cancelTemporaryOverride(OverrideCancelAction)
+    case bolusEntry(BolusAction)
+    case carbsEntry(CarbAction)
```

Comment:
REMOTE 2.0: WILL REMOVE PRE-MERGE

Referring to the ClosedLoopAction and AutobolusAction parts of this class.